scripts/runTests.mjs

```diff
@@ -4,28 +4,26 @@
  *--------------------------------------------------------------------------------------------*/
 import { spawn } from 'node:child_process';
 
-try {
-  const argument = process.argv[2];
+const argument = process.argv[2];
 
-  if (argument?.includes('filter')) {
-    spawn(`turbo run test ${argument}`, {
-      stdio: 'inherit',
-      shell: true,
-    });
-  } else if (argument) {
-    spawn(
-      `conc -g -n "CSS-workshop,Unit,A11y,React-workshop,e2e" -c "#8BCC2F,#D8598C,#539AD0,#D1B52A,white" "pnpm test:css --filter=${argument.toLowerCase()}" "pnpm test:unit ${argument}" "pnpm test:a11y --env componentName=${argument}" "pnpm test:react --spec="**/${argument}.*"" "pnpm run test:e2e -- ${argument}"`,
-      {
-        stdio: 'inherit',
-        shell: true,
-      },
-    );
-  } else {
-    spawn(`turbo run test`, {
-      stdio: 'inherit',
-      shell: true,
-    });
-  }
-} catch (error) {
-  console.log(error);
+if (argument?.includes('filter')) {
+  startProcess(`turbo run test ${argument}`);
+} else if (argument) {
+  startProcess(
+    `conc -g -n "CSS-workshop,Unit,A11y,React-workshop,e2e" -c "#8BCC2F,#D8598C,#539AD0,#D1B52A,white" "pnpm test:css --filter=${argument.toLowerCase()}" "pnpm test:unit ${argument}" "pnpm test:a11y --env componentName=${argument}" "pnpm test:react --spec="**/${argument}.*"" "pnpm run test:e2e -- ${argument}"`,
+  );
+} else {
+  startProcess(`turbo run test`);
+}
+
+function startProcess(command) {
+  const subprocess = spawn(command, {
+    stdio: 'inherit',
+    shell: true,
+  });
+  subprocess.on('exit', (code) => {
```

Comment:
I think instead of listening to the `exit` event, you might be able to instead use [`spawnSync`](https://nodejs.org/api/child_process.html#child_processspawnsynccommand-args-options) and use the `status` from its return value

Reply:
That does look like it should work. I'll work on implementing it.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1991/commits/1358882931ae676dfc38781c3b138f55acaa18aa)